### ISSUE TITLE
Podman does not ship systemd units for APIv2

### ISSUE THREAD

**Issue by marusak** [containers/libpod#5563]
**Is this a BUG REPORT or FEATURE REQUEST? (leave only one on its own line)**

/kind bug

**Description**
The new release `podman-1.8.2-1` is still not shipping systemd units for the new APIv2 in Fedora.
It seems it exists in the repository in `contrib/systemd/system/podman.service` though.

**Steps to reproduce the issue:**
```
$ rpm -ql podman | grep systemd
/usr/lib/systemd/system/io.podman.service
/usr/lib/systemd/system/io.podman.socket
/usr/lib/systemd/user/io.podman.service
/usr/lib/systemd/user/io.podman.socket
```
These are though units for varlink and not for the REST API.

**Package info (e.g. output of `rpm -q podman` or `apt list podman`):**
```
podman-1.8.2-1.fc31.x86_64
```

**Comment by rhatdan** [comment]
I have patched the spec files and retriggered builds so we can get this release updated without having to bump the release.

**Comment by marusak** [comment]
I guess this was fixed in #5567
